From ce06f2a6f0509568a1c9aef345526996949e1fc0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=23R=CE=BBatan?= <sharlatanus@gmail.com>
Date: Sat, 12 Dec 2020 00:21:24 +0000
Subject: [PATCH] gnu/packages/lisp-xyz.scm

Add new variables: sbcl-command-line-arguments cl-command-line-arguments ecl-command-line-arguments
---
 gnu/packages/lisp-xyz.scm | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index f77e949fea..d596811aab 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -144,6 +144,35 @@ In order to make one or several variables file-local, use the macros
 (define-public ecl-net.didierverna.asdf-flv
   (sbcl-package->ecl-package sbcl-net.didierverna.asdf-flv))
 
+(define-public sbcl-command-line-arguments
+  (let ((commit "fbac862fb01c0e368141204f3f639920462c23fe")
+        (revision "1"))
+  (package
+   (name "sbcl-command-line-arguments")
+   (version (git-version "2.0.0" revision commit))
+   (source (origin
+            (method git-fetch)
+            (uri (git-reference
+                  (url "https://github.com/fare/command-line-arguments")
+                  (commit commit)))
+            (file-name (git-file-name name version))
+            (sha256
+             (base32
+              "054m1ikndzqf72mb9ajaa64136cwr3bgag4yfbi1574a9vq75mjq"))))
+   (build-system asdf-build-system/sbcl)
+   (home-page "https://github.com/fare/command-line-arguments")
+   (synopsis "Trivial command-line argument parsing library for Common Lisp")
+   (description "A library to abstract away the parsing of Unix-style
+command-line arguments.  Use it in conjunction with asdf:program-op or cl-launch
+for portable processing of command-line arguments.")
+   (license license:expat))))
+
+(define-public ecl-command-line-arguments
+  (sbcl-package->ecl-package sbcl-command-line-arguments))
+
+(define-public cl-command-line-arguments
+  (sbcl-package->cl-source-package sbcl-command-line-arguments))
+
 (define-public sbcl-fiveam
   (package
     (name "sbcl-fiveam")
-- 
2.26.2

